lib/tasks/link_checker.rake

```diff
@@ -30,15 +29,22 @@ task :link_checker, [:date] => [:environment] do |_, _args|
       other_urls << url
     end
   end
+  [gov_urls, other_urls]
+end
+
+desc "link checker"
+task :link_checker, [:date] => [:environment] do |_, _args|
+  govuk_urls, other_urls = get_urls_from_locale_files
+
   link_checker = GdsApi::LinkCheckerApi.new(
     "https://link-checker-api.publishing.service.gov.uk",
     bearer_token: ENV["LINK_CHECKER_API_BEARER_TOKEN"],
   )
 
-  puts "LINKS TO CHECK #{urls.length}"
+  puts "LINKS TO CHECK #{govuk_urls.length + other_urls.length}"
 
   link_report_batch = link_checker.create_batch(
-    urls,
+    govuk_urls + other_urls,
```

Comment:
As these two lists are being concatenated anyway, can `get_urls_from_locale_files` return one list of urls?